include_core/omrport.h

```diff
@@ -1012,6 +1012,13 @@ typedef struct OMROSKernelInfo {
 #define OMR_CGROUP_SUBSYSTEM_MEMORY ((uint64_t)0x2)
 #define OMR_CGROUP_SUBSYSTEM_ALL (OMR_CGROUP_SUBSYSTEM_CPU | OMR_CGROUP_SUBSYSTEM_MEMORY)
 
+typedef enum OMRContainerType {
+	OMR_CONTAINER_NONE,
+	OMR_CONTAINER_DOCKER,
```

Comment:
I checked `runc` container and it displays same behavior with respect to `/proc/pid/cgroup` file. So I believe the detection mechanism should work for any `runc`-style container, not just Docker.
Since Docker is most popular system at the moment, I named it so.
Do you want to rename it as something like `OMR_CONTAINER_RUNC`?